### ISSUE TITLE
Cannot handle the struct when the type hierarchy is too deep.

### ISSUE THREAD

**Issue by pCYSl5EDgo** [Cysharp/csbindgen#15]
I've installed [libpng](http://www.libpng.org/pub/png/libpng.html) via [vcpkg](https://github.com/microsoft/vcpkg).

The problem is that struct `png_row_info_struct`, which `bindgen` did generate, is not generated by `csbindgen`.

The following source code is about as much as we can reproduce the problem.

#### build.rs

```rs:build.rs
use std::error::Error;

fn main() -> Result<(), Box<dyn Error>> {
    // bindgen::Builder::default()
    //     .header("C:/src/vcpkg/installed/x64-windows/include/libpng16/png.h")
    //     .header("C:/src/vcpkg/installed/x64-windows/include/libpng16/pngconf.h")
    //     .header("C:/src/vcpkg/installed/x64-windows/include/libpng16/pnglibconf.h")
    //     .generate()
    //     .unwrap()
    //     .write_to_file("src/libpng16.rs")
    //     .unwrap();

    csbindgen::Builder::new()
        .input_bindgen_file("src/libpng16.rs")
        //.method_filter(|x| x.starts_with("png_"))
        .csharp_namespace("PixivApi.ImageFile")
        .csharp_class_name("LibPng16")
        .csharp_dll_name("libpng16")
        .generate_csharp_file("../src/PixivApi.ImageFile/libpng16_csbindgen.cs")?;

    Ok(())
}
```

[The result files gist](https://gist.github.com/pCYSl5EDgo/dec061f211855110c3d9eaaead637db6)

**Comment by neuecc** [comment]
Thank you, the issue was that I wasn't searching for the required types within the parameters of the type alias function.
I have now fixed this by including the parameters of functions within type aliases in the search scope.
Fix is included in `v1.5.0`.

**Comment by pCYSl5EDgo** [comment]
Thanks!
